app/views/requests/_letter_of_consent.html.erb

```diff
@@ -1,5 +1,10 @@
-<%= f.govuk_file_field :letter_of_consent, label: nil, hint: { text: simple_format(t("helpers.hint.request_form.letter_of_consent")) } %>
-<p class="govuk-body-m"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>
+<span class="govuk-body">You must provide a letter of consent from the subject of this request which:</span>
```

Comment:
A `span` is an inline element. If you look at other forms in this app you'll see they use a `p` tag which is a block element when additional text is required.